src/utils/SetupUtil.js

```diff
@@ -82,6 +82,9 @@ export default {
           machineVersion
         });
 
+        virtualBox.getShareDir(machine.name());
```

Comment:
If a user adds a shared directory, it will only be seen until the next kitematic restart - Is this the correct approach?